src/components/MessageList/MessageList.tsx

```diff
@@ -210,6 +210,8 @@ export type MessageListProps<
    * Defaults to and accepts same props as: [TypingIndicator](https://getstream.github.io/stream-chat-react-native/#typingindicator)
    */
   TypingIndicator?: React.ComponentType<TypingIndicatorProps>;
+  /** Whether or not the FlatList is inverted. Defaults to true */
+  invertedList?: boolean;
```

Comment:
could you throw this in alphabetical order within the types as the rest of them are?
```suggestion
  /** Whether or not the FlatList is inverted. Defaults to true */
  inverted?: boolean;
```

Reply:
also add a `FooterComponent` just like exists for `HeaderComponent` so whichever one wants to be used can be used instead of basing it off of the `inverted` flag. showed the corresponding change below. you'll also need to change the Thread component name from `headerComponent` -> `footerComponent` and change the usage when passed into MessageList from `HeaderComponent={headerComponent}` to `FooterComponent={footerComponent}`